content/en/docs/collector/configuration.md

```diff
@@ -627,6 +627,24 @@ service:
       processors: [batch, memory_limiter]
       exporters: [opencensus, zipkin]
 ```
+As with components, use the `type[/name]` syntax to create additional pipelines for a given type.
+
+```yaml
+service:
+  pipelines:
+    metrics:
+      receivers: [opencensus, prometheus]
+      processors: [batch]
+      exporters: [opencensus, prometheus]
+    traces:
+      receivers: [opencensus, jaeger]
+      processors: [batch, memory_limiter]
+      exporters: [opencensus, zipkin]
+    traces/better_pipeline:
+      receivers: [opencensus]
+      processors: [batch]
+      exporters: [zipkin]
+```
```

Comment:
How about this so that we can keep the focus on what was added?
~~~suggestion
```yaml
service:
  pipelines:
    # ...
    traces:
      # ...
    traces/better_pipeline:
      receivers: [opencensus]
      processors: [batch]
      exporters: [zipkin]
```
~~~

Reply:
Also, is there an alternative name to `/better_pipeline` that we can use? Previous examples in the page use `/2`.

Reply:
done :)